lib/AST/FrontendSourceFileDepGraphFactory.cpp

```diff
@@ -282,7 +282,8 @@ DependencyKey DependencyKey::createDependedUponKey(StringRef mangledHolderName,
 
 bool fine_grained_dependencies::emitReferenceDependencies(
     DiagnosticEngine &diags, SourceFile *const SF,
-    const DependencyTracker &depTracker, StringRef outputPath,
+    const DependencyTracker &depTracker,
+    const ReferencedNameTracker *nameTracker, StringRef outputPath,
```

Comment:
Better to pass in a bool instead of a whole pointer, if my supposition is right.